modules/core/server/controllers/core.server.controller.js

```diff
@@ -57,3 +58,11 @@ exports.receiveCSPViolationReport = function(req, res) {
   }
   res.status(204).json();
 };
+
+/**
+* Render javascript content containing service worker config.
+*/
+exports.renderServiceWorkerConfig = function(req, res) {
+  res.set('Content-Type', 'text/javascript')
+     .send('var FCM_SENDER_ID = ' + JSON.stringify(config.fcm.senderId) + ';\n');
+};
```

Comment:
Express has support for [jsonp](https://expressjs.com/en/api.html#res.jsonp), but normally configs have been exposed to index html template from [config/lib/express.js](https://github.com/Trustroots/trustroots/blob/master/config/lib/express.js#L31).